docs/ChainOrderTransitivity.md

```diff
@@ -0,0 +1,153 @@
+      This is because transitivity is so ingrained when thinking about orders that it is hard to not implicitly assume.
+    - Certain "obvious" properties suddenly fail to hold, for example:
+
+      > Repeatedly observing the selection of a node over a period of time yields a sequence that is strictly improving for every two adjacent selections.
+
+      In particular, this might have practical effects for diffusion pipelining, which requires such a condition.
+
+## Possible solutions
+
+One solution is to change the chain order in such a way that it is transitive and still has (mostly) the same "features" as before.
+
+### Regarding opcert numbers
+
+There is a simple fix for the intransitivity caused by the opcert numbers: swap the orders in which opcert numbers and VRFs are compared, back to what it was before [ouroboros-network#2348](https://github.com/IntersectMBO/ouroboros-network/pull/2348).
+The resulting is transitive, hence total.
+
+The motivation for the opcert number tiebreaker is described in the ["Design Specification for Delegation and Incentives in Cardano"][ledger-readme].
+We describe our understanding of this use case for the opcert number tiebreaker:
+
+Consider an attacker who corrupts an SPO's node with a specific issuer identity, including the hot key, but not the cold key.
+They can now forge blocks on behalf of this issuer identity, deciding which transactions are included in the block, potentially spoiling the reputation of the SPO in the process.
+(Rewards still go to the account of the issuer identity, which the attacker can not access.)
+The attacked SPO can now issue a new hot key with a higher opcert number, and start a new node with it installed.
+Now, in every slot where this issuer identity is elected, both the attacker and the attacked SPO can forge a block.
+
+ - If the SPO forges a strictly longer or shorter chain, the opcert tiebreaker does not matter, citing from the aforementioned specification:
+   > Note that nodes take the precedence amongst operational key certificates into account only after comparing the length of the chains.
+   > When the node is already up to date and receives two conflicting blocks that add to its current chain, the length will of course always be the same.
+   > But this rule is important: if we did not compare the lengths of the chains before giving preference to the block with the newer operational certificate, it would be possible to force a node to do a rollback of arbitrary length, by sending it a block from a past slot, signed using a newer certificate than the block that the node already has in its chain for that slot.
+   > This would open up an attack where a stake pool operator could force nodes to do arbitrary rollbacks.
+ - If the SPO forges on top of a chain that is just as long as the one the attacker forged on, the opcert number tiebreaker gives precedence to the SPO as they have the higher opcert number.
```

Comment:
"If the SPO and attacker instead mint chains of the same length," seems more direct